n2y/blocks.py

```diff
@@ -570,8 +570,30 @@ def to_pandoc(self):
         return self.children_to_pandoc()
 
 
-class LinkToPageBlock(WarningBlock):
-    pass
+class LinkToPageBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
+
+        self.link_type = self.notion_data["type"]
+        # The key for the object id may be either "page_id"
+        # or "database_id".
+        self.linked_page_id = self.notion_data[self.link_type]
+
+    def to_pandoc(self):
+        # TODO: in the future, if we are exporting the linked page too, then add
+        # a link to the page. For now, we just display the text of the page.
+
+        # The linked page may be outside the scope of what is accessible to n2y.
+        try:
+            page = self.client.get_page_or_database(self.linked_page_id)
+            title = page.title.to_pandoc()
+        except PermissionError:
+            msg = (
+                "Permission denied when attempting to access linked page having id [%s]"
+            )
+            logger.warning(msg, self.linked_page_id)
```

Comment:
Please see the comment on my previous review here: https://github.com/innolitics/n2y/pull/69#discussion_r967007485

I don't think you fully addressed the comment.